web/src/form/form-section.html

```diff
@@ -13,6 +13,10 @@
             type: String,
             value: "",
           },
+          label: {
+            type: String,
+            value: ""
+          }
```

Comment:
interesting, i see label here but [[name]] below, perhaps change label to name here?